store/storetest/session_store.go

```diff
@@ -116,19 +116,15 @@ func testSessionRemove(t *testing.T, ss store.Store) {
 	s1, err := ss.Session().Save(s1)
 	require.Nil(t, err)
 
-	if session, err := ss.Session().Get(s1.Id); err != nil {
-		t.Fatal(err)
-	} else {
-		if session.Id != s1.Id {
-			t.Fatal("should match")
-		}
-	}
+	session, err := ss.Session().Get(s1.Id);
+	require.Nil(t, err)
+	require.Equal(t, s1.Id, session.Id)
 
-	store.Must(ss.Session().Remove(s1.Id))
+	err = ss.Session().Remove(s1.Id)
+	require.Nil(t, err)
 
-	if _, err := ss.Session().Get(s1.Id); err == nil {
-		t.Fatal("should have been removed")
-	}
+	_, err = ss.Session().Get(s1.Id)
+	require.Error(t, err)
```

Comment:
it should be `require.NotNil` we are using `*model.AppError` which is compatible with the `error` interface, but is not an error itself, so... We use `require.Nil` and `require.NotNil`.